hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/README.md

```diff
@@ -62,3 +67,21 @@ YARN UI has replaced NPM with Yarn package manager. And hence Yarn would be used
 ### Adding new routes (pages), controllers, components etc.
 
 * Use ember-cli blueprint generator - [Ember CLI](http://ember-cli.com/extending/#generators-and-blueprints)
+
+### Building with Maven
+
+[YARN-6278](https://issues.apache.org/jira/browse/YARN-6278)
+added build profile using frontend-maven-plugin which
```

Comment:
"a" build profile.

What is "frontend-maven-plugin"? I think this looks more like:
"YARN-6278 added a build profile to frontend-maven-plugin which automatically..."

Reply:
added link to frontend-maven-plugin as a reference.